### PR TITLE
Amplitude addition disk

### PR DESCRIPTION
**Pull request by korgan00**
Fix https://github.com/Qiskit/qiskit.org/issues/2292

### CODE REVIEW

File: frontend/vue/components/AmplitudeDisk/AmplitudeAdditionWidget.vue [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    this.highlightedA = false
+    this.highlightedB = false
+  },
+  methods: {
+    updateInternalAmplitudeA (amplitude: Amplitude) {
+      if (this.highlightTimeoutA !== -1) {
+        clearTimeout(this.highlightTimeoutA)
+      }
+      this.highlightedA = true
+      this.highlightTimeoutA = setTimeout(this.removeHighlightA, 1000)
```
Comment (vabarbosa) [inline frontend/vue/components/AmplitudeDisk/AmplitudeAdditionWidget.vue:91]: @korgan00 there are some build failures (related to Typescript) with the various timeout variables. for example:

`TS2322: Type 'Timeout' is not assignable to type 'number'.`

**Review by vabarbosa — CHANGES_REQUESTED** [review]
there are some build failures (related to Typescipt) with the Timeout variables

File: frontend/vue/components/AmplitudeDisk/AmplitudeAdditionWidget.vue (hunk 1 above)
Comment (korgan00) [reply frontend/vue/components/AmplitudeDisk/AmplitudeAdditionWidget.vue:91]: Sorry, it was working with the auto-refresh in dev.
The issue is related to node. It has an inner setTimeout and it has a different return type than the browser one.
It should be fixed now.

**Review by vabarbosa — APPROVED** [review]
thank you

**Review by vabarbosa — APPROVED** [review]

**Pull request merged**